oojs.js

```diff
@@ -10,3 +10,37 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+//Second attempt
+class Playlist{
+ constructor(pltitle){
+   this.pltitle = title;
+   this.songtitle = [];
+ }
```

Comment:
stylistic notes: `plTitle`,`songTitle`
constructor should take a second arg, `songs` which you should assume is an array of strings (songs represented as string). Everything else you NAILED. Nice work.